src/main/java/org/fundacionjala/coding/KevinHerrera/Movies/Main.java

```diff
@@ -0,0 +1,31 @@
+package org.fundacionjala.coding.KevinHerrera.Movies;
+
+import org.fundacionjala.coding.KevinHerrera.Movies.Movie.Movie;
+import org.fundacionjala.coding.KevinHerrera.Movies.Movie.MovieFactory;
+import org.fundacionjala.coding.KevinHerrera.Movies.Customer.Customer;
+
+/**
+ * @author KevinHerrera - AWT-[01].
+ * @version 0.1
+ */
+
+public class Main {
+    /**
+     * .
+     */
+    protected Main() {
+    }
+
+    /**
+     * main class.
+     * @param args args
+     */
+    public static void main(final String[] args) {
+        Customer customer = new Customer("Test");
+        Movie movieRevenant = MovieFactory.createNewReleaseMovie("The Revenant");
+        Movie movieTerminator = MovieFactory.createRegularMovie("Terminator");
+        customer.rentMovie(movieRevenant, 2);
+        customer.rentMovie(movieTerminator, 2);
+        System.out.println(customer.statement());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)